tensorflow/tools/compatibility/tf_upgrade_v2_main.py

```diff
@@ -85,7 +87,7 @@ def main():
       raise ValueError(
           "--outfile=<output file> argument is required when converting a "
           "single file.")
-    files_processed, report_text, errors = upgrade.process_file(
+    files_processed, report_text, errors = ipynb.process_file(
```

Comment:
I'd prefer the filename selection logic to live here.

Reply:
Also, look at line 99, we should make sure that convert tree also understands ipynb files.